### ISSUE TITLE
Theme customization form fields with default value 0 not working

### ISSUE THREAD

**Issue by JumpLink** [octobercms/october#2388]
Theme customization form fields with default value 0 (and false) are not working.
##### Expected behavior

A form field like

``` yaml
            bs4-grid-breakpoints-xs:
                type: number
                label: grid-breakpoints-xs
                default: 0
                tab: Bootstrap 4.0.0-alpha.4
                assetVar: grid-breakpoints-xs
```

should be 0 in the backend under theme customization.
##### Actual behavior

bs4-enable-flex is empty / not set

After I change this variable it works (also if I set it to 0 in the backend) but not before with his default value.
##### Reproduce steps

Create a fields in your theme.yaml file like

```
form:
    tabs:
        fields:
            bs4-grid-breakpoints-xs:
                type: number
                label: grid-breakpoints-xs
                default: 0
                tab: Bootstrap 4.0.0-alpha.4
                assetVar: grid-breakpoints-xs
```

and check out the results at theme customization, it will be empty.
##### October build

365

**Comment by leocavalcante** [comment]
Have you tried setting it between quotes?
`default: '0'`

**Comment by JumpLink** [comment]
Yes, is not working, too

**Comment by daftspunk** [comment]
I was able to replicate this.

**Comment by LukeTowers** [comment]
Does somebody want to test #2859 and let me know if it solves the problem?

**Comment by TimFoerster** [comment]
@LukeTowers It works for me

**Comment by LukeTowers** [comment]
Fixed by #2859
